src/legacy/core_plugins/kibana/public/home/np_ready/components/tutorial/tutorial.js

```diff
@@ -363,6 +363,11 @@ class TutorialUi extends React.Component {
         );
       }
 
+      let icon = this.state.tutorial.euiIconType;
+      if (icon != null && icon.includes('/')) {
```

Comment:
Please don't use type unsafe comparison operators (`!=`). Please either use `!==` or just a truthy check on the variable instead (if we don't know if it's `null` or `undefined`).